docs/source/readers.rst

```diff
@@ -0,0 +1,126 @@
+=========================
+Readers
+=========================
+
+.. warning::
+    This document is not complete !
+    
+    Currently under development. 
+    
+    Do not use it now. 
+
+DISDRODB supports reading and loading data from many input file formats and schemes. The following sections describe the different way data can be loaded, requested, or added to the DISDRODB project.
+
+
+
+Available Readers
+==================
+
+.. note::
+    RL : Could be nice to have a function that returns all readers 
+
+
+.. code-block:: bash
+
+	from disdrodb import available_readers
+	available_readers()
+	
+
+
+Add a new reader
+==================
+
+
+.. note::
+    RL : Here we should prepare a fake project with sample data to give example for every steps. It should be a very light and simple dataset. 
+
+
+
+**Step 1 : Create the file structure for the reader.**
+
+Go under the reader folder and add yours (or use an exsiting one).
+
+This folder must be named with the name of this institution. 
+
+.. note::
+    RL : we should formalize that ? How to name the folders to have consistency. Currently, there are institution names and country names. 
+
+
+
+
+**Step 2 : Get the template python scripts.**
+
+Copy paste the templates/l0_parser_dev_template into the folder proviously created. 
+
+This file will be used as basis for your devlopment. 
+
+
+
+**Step 3 : Prepare the raw and processes file tree.** 
+
+The input raw folder tree must correspond to : 
+
+| 📁 DISDRODB/
+| ├── 📁 Raw/
+|    ├── 📁 Name of the institution/

```

Comment:
Here currently is institution or country. 
We use the `institution` name when the campaigns data spans more than 1 country.
We use `country` when all campaigns (or sensor network) are inside a given country.

Reply:
ok